glide-core/src/protobuf/redis_request.proto

```diff
@@ -199,6 +199,8 @@ enum RequestType {
     XDel = 166;
     LMove = 168;
     BLMove = 169;
+    BitField = 170;
+    BitFieldReadOnly = 171;
```

Comment:
I think we should just keep it identical to the command
```suggestion
    BitField_Ro = 171;
```

Reply:
Not sure I agree with this, it doesn't seem java like to have camel case and underscore and ignore the acronym. Jedis names it similar for example https://javadoc.io/doc/redis.clients/jedis/4.2.3/redis/clients/jedis/commands/StringCommands.html#bitfieldReadonly-java.lang.String-java.lang.String...-.

Thoughts?